content/packages-and-modules/contributing-packages-to-the-registry/about-package-readme-files.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: About package README files
+title: About package README.md files
```

Comment:
They do not have to be `.md` files.  The file name can just be `README`.

The title is probably best to leave as-is.

Prescribing `README.md` below in the actual copy below is good still though.

Reply:
Changed for the sake of consistency.
README files - in my limited experience - are always in markdown format just as package is in JSON format i.e `package.json`

Reply:
A kind reminder that I await for your input on the matter.